pick.lic

```diff
@@ -221,8 +222,11 @@ class LockPicker
     bput("get my #{box} from my #{@settings.picking_box_source}", 'You get a .* from inside ')
 
     unless disarm?(box)
-      if @settings.picking_box_storage
-        bput("put my #{box} in my #{@settings.picking_box_storage}", 'You put your .* in ') if right_hand
+      if @settings.picking_box_storage && right_hand
+		case bput("put my #{box} in my #{@settings.picking_box_storage}", 'You put your .* in ', "You just can\'t get", "There isn\'t any more")
```

Comment:
Very large indentation here. I try to keep things to a 2 space tab. This is pretty standard in the industry.